spec/fixtures/pages/storage/local_storage.html

```diff
@@ -1,11 +1,8 @@
 <script>
-  const {ipcRenderer} = require('electron')
-
-  let message = 'Ok'
   try {
     window.localStorage
   } catch (e) {
-    message = e.message
+    const {ipcRenderer} = require('electron')
+    ipcRenderer.send('local-storage-response', e.message)
   }
```

Comment:
This looks like it won't send a message back if it doesn't throw, which will just make the test time out. Let's make tests fail fast & with useful error messages rather than dealing with yet another "test timed out" failure.